test/embind/embind.test.js

```diff
@@ -642,6 +642,10 @@ module({
             assert.throws(TypeError, function() { cm.const_ref_adder(0n, 1); });
         });
 
+        // The test harness should set assertions to either be enabled, or not;
+        // this value should not be unset.
+        assert.true(cm['ASSERTIONS'] !== undefined);
```

Comment:
What about the other two settings we reflect out onto the module?

Should we instead use `=== true` everywhere?

Reply:
I added the other two.

What do you mean by `=== true`? (The value here can be `true` or `false`, so the assertions just check it was set and is not undefined.)

Reply:
I mean rather than use truthy tests like `if (cm["X"])` we could use `if (cm["X"] === true)`, and then we wouldn't need that assertion, but either way seems fine.

Reply:
Now I see, thanks.

I think it's more verbose to do `=== true` and then `=== false` and then have an else for the error code. Just adding these assertions seems better to me.